tests/medium/test_ide.py

```diff
@@ -411,6 +411,27 @@ def test_install_with_changed_download_page(self):
         self.assertFalse(self.is_in_path(self.exec_link))
 
 
+class SublimeTextInContainer(ContainerTests, test_ide.SublimeTextTests):
+    """This will test the Sublime Text integration inside a container"""
+
+    TIMEOUT_START = 20
+    TIMEOUT_STOP = 10
+
+    def setUp(self):
+        self.hosts = {443: ["sublimetext.com", "download.sublimetext.com"]}
+        super().setUp()
+        # override with container path
+        self.installed_path = os.path.join(self.install_base_path, "ide", "sublime-text")
+
+    def test_install_with_changed_download_page(self):
```

Comment:
You don't need this one as you are using the base parser in BaseInstaller. There is already a test in BaseInstaller to ensure we support bad page parsing :)